test/cypress/integration/settings/switch-psp-onboarding-cy.test.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const userStubs = require('../../stubs/user-stubs')
+const gatewayAccountStubs = require('../../stubs/gateway-account-stubs')
+const stripeAccountSetupStubs = require('../../stubs/stripe-account-setup-stub')
+
+const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
+const gatewayAccountId = '42'
+const gatewayAccountExternalId = 'a-valid-external-id'
+
+
+function getUserAndAccountStubs (paymentProvider , providerSwitchEnabled, gatewayAccountCredentials, merchantDetails,) {
+  return [
+    userStubs.getUserSuccess({ gatewayAccountId, userExternalId, merchantDetails }),
+    gatewayAccountStubs.getGatewayAccountByExternalIdSuccess({ gatewayAccountId, gatewayAccountExternalId, providerSwitchEnabled, paymentProvider, ...gatewayAccountCredentials && { gatewayAccountCredentials } })
+  ]
+}
+
+describe('Switch PSP settings page', () => {
```

Comment:
This is not a switch PSP (we are only following the pattern). This should be named after the functionality being tested